Client/game_sa/CWeatherSA.h

```diff
@@ -26,4 +26,39 @@ class CWeatherSA : public CWeather
     float GetAmountOfRain();
     void  SetAmountOfRain(float fAmount);
     void  ResetAmountOfRain();
+
+    float GetWetRoads();
+    bool  SetWetRoads(float fAmount);
+    bool  ResetWetRoads();
+
+    float GetFoggyness();
+    bool  SetFoggyness(float fAmount);
+    bool  ResetFoggyness();
+
+    float GetFog();
+    bool  SetFog(float fAmount);
+    bool  ResetFog();
+
+    float GetRainFog();
+    bool  SetRainFog(float fAmount);
+    bool  ResetRainFog();
+
+    float GetWaterFog();
+    bool  SetWaterFog(float fAmount);
+    bool  ResetWaterFog();
+
+    float GetSandstorm();
+    bool  SetSandstorm(float fAmount);
+    bool  ResetSandstorm();
+
+    float GetRainbow();
+    bool  SetRainbow(float fAmount);
+    bool  ResetRainbow();
+
+private:
+    static unsigned char* VAR_CWeather__ForcedWeatherType;
+    static unsigned char* VAR_CWeather__OldWeatherType;
+    static unsigned char* VAR_CWeather__NewWeatherType;
```

Comment:
`std::uint8_t` instead of `unsigned char`

Reply:
By the way, I'm not the author of these changes. It was merged from master in a1db8f4 (it looks like some private changes, I don't really know where it originated in and who is the author). It's not even used in public files as far as I know.

Reply:
> By the way, I'm not the author of these changes. It was merged from master in [a1db8f4](https://github.com/multitheftauto/mtasa-blue/commit/a1db8f45913c7bc4b4f7c76fcd0163c9495961f1) (it looks like some private changes, I don't really know where it originated in and who is the author). It's not even used in public files as far as I know.

But a refactor in this case would be better